<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-0356DV5C" ignored="0" name="t2" tests="1" failures="1" timestamp="02 五月 2022 10:33:32 GMT" time="6.249" errors="0">
  <testcase name="test111" time="0.006" classname="com.webtest.testCases.test1">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]at com.webtest.testCases.test1.test111(test1.java:14)... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test111 -->
</testsuite> <!-- t2 -->
